### PR TITLE
Updates ellipsometry manual page

### PR DESCRIPTION
**Pull request by domna**

### CODE REVIEW

File: manual/source/classes/contributed_definitions/ellipsometry-structure.rst [hunk 1]
```diff
@@ -14,42 +14,22 @@ Optical Spectroscopy
...
-       Added fields to describe spectroscopic detection with polarization (e.g. rotating analyzer).
-       
-    :ref:`NXaperture`
-       Added fields to define parameters that describe windows (e.g. windows of a UHV cryostat), such as the thickness and the orientation angle of the window, as well as reference data to calculate window effects.
-       
-    :ref:`NXsample`
-       Added fields to specify the sample and material properties, as well as the sample environment (e.g. refractive index of surrounding medium) and experimental conditions (e.g. temperature, pressure, pH value etc.).
+       An application definition for ellipsometry measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
 
-.. _DispersiveMaterial:
```
Comment (sanbrock) [inline manual/source/classes/contributed_definitions/ellipsometry-structure.rst:52]: we need the index as it is defined above

**Review by sanbrock — CHANGES_REQUESTED** [review]
do not we have any new base classes?

**Comment by domna** [comment]
> do not we have any new base classes?

Yes, there are some for the dispersive material which are stated below, or are you referring to other base classes?

File: manual/source/classes/contributed_definitions/ellipsometry-structure.rst (hunk 1 above)
Comment (domna) [reply manual/source/classes/contributed_definitions/ellipsometry-structure.rst:52]: Not sure what you are referring to

**Comment by sanbrock** [comment]
NXbeam_path, NXenvironment,, NXprogram, NXlens_opt, NXwaveplate - I think these are our new base classes which need to be listed similarly to those of dispersions'

File: manual/source/classes/contributed_definitions/ellipsometry-structure.rst (hunk 1 above)
Comment (sanbrock) [reply manual/source/classes/contributed_definitions/ellipsometry-structure.rst:52]: this line is an anchor which we shall not remove to enable url to point directly to this subsection

Comment (domna) [reply manual/source/classes/contributed_definitions/ellipsometry-structure.rst:52]: aah I see. This was an accident actually...

**Comment by domna** [comment]
> NXbeam_path, NXenvironment,, NXprogram, NXlens_opt, NXwaveplate - I think these are our new base classes which need to be listed similarly to those of dispersions'

Added

**Review by sanbrock — APPROVED** [review]

**Pull request merged**